css/css.go

```diff
@@ -70,7 +70,8 @@ func (c *cssMinifier) minifyGrammar() error {
 	semicolonQueued := false
 	for {
 		gt, _, data := c.p.Next()
-		if gt == css.ErrorGrammar {
+		switch gt {
```

Comment:
I thought there might be value in using switch blocks consistently here and in the following switch block because we're doing the same thing in matching against `gt`

Reply:
Fair enough. I also think the second case can end with a `continue` though, as those cases are not matched by the second switch statement.